OPHD/UI/Core/Label.h

```diff
@@ -39,6 +39,5 @@ class Label : public TextControl
 
 private:
 	NAS2D::Color textColor{NAS2D::Color::White};
-	const NAS2D::Font* mFont;
```

Comment:
This looks to be an accidental change. The commit message doesn't match up with the change.

Reply:
Yeah, that was an accidental delete. Will fix shortly.